src/plugins/discover/server/ui_settings.ts

```diff
@@ -315,7 +315,7 @@ export const getUiSettings: (docLinks: DocLinksServiceSetup) => Record<string, U
     value: false,
     description: i18n.translate('discover.advancedSettings.enableSQLDescription', {
       defaultMessage:
-        '{technicalPreviewLabel} This tech preview feature is highly experimental--do not rely on this for production saved searches or dashboards. This setting enables SQL as a text-based query language in Discover. If you have feedback on this experience please reach out to us on {link}',
+        '{technicalPreviewLabel} This tech preview feature is highly experimental--do not rely on this for production saved searches, visualizations or dashboards. This setting enables SQL as a text-based query language in Discover and Lens. If you have feedback on this experience please reach out to us on {link}',
```

Comment:
ℹ️ We are using the same switch with discover. I have slightly changed the text to include Lens. The setting is still on the Discover section.